Src/WindowsServer/WindowsServer.Shared/FirstChanceExceptionStatisticsTelemetryModule.cs

```diff
@@ -220,13 +220,22 @@ private void TrackStatistics(string type, string operation, string method, bool
 
         private string GetDimCappedString(string value, ConcurrentDictionary<string, int> capValues)
         {
-            if (capValues.Count > 100)
+            int tmp;
+            var result = capValues.TryGetValue(value, out tmp);
+
+            if (result)
+            {
+                return value;
+            }
+            else if (capValues.Count > 100)
```

Comment:
we do not need strong capping, just preventing from hitting ourselves in a foot. And I also wait for proper built-in functionality of dim capping in Metric Aggregation